fmriprep/workflows/bold/resampling.py

```diff
@@ -356,6 +407,8 @@ def init_bold_preproc_trans_wf(mem_gb, omp_nthreads,
 
     **Parameters**
 
+        freesurfer : bool
+            Enable sampling of FreeSurfer files
```

Comment:
This parameter does not appear in the function definition. I think perhaps it was supposed to be in the `init_bold_mni_trans_wf` docstring.

Reply:
yes. moved the comments to the correct docstring